pages/languages/_static/c-sharp.svg

```diff
@@ -12,8 +12,8 @@
    version="1.1"
    id="svg2"
    xml:space="preserve"
-   width="64px"
-   height="64px"
+   width="82px"
```

Comment:
64 is a nice width. Do you think it would make more sense to just use CSS to make the image fill the container?